.travis.yml

```diff
@@ -23,12 +25,14 @@ script:
   - test -z "$(go vet $(find ./arm/* -type d -print) | tee /dev/stderr)"
   - test -z "$(golint ./arm/... | tee /dev/stderr)"
   - go build ./arm/examples/...
-  - go test -v ./management/...
+  - if [[ $TRAVIS_GO_VERSION != 1.6* ]]; then
```

Comment:
@mcardosos I will update the README for ASM to include a note about requiring Go 1.7+ (this dependency was introduced in commit 2cb6120 and since it's legacy I don't think it's worth investing the effort to get it working on 1.6 again).
@marstr What is the error that will happen in this case?